doc/ref_model/chapters/chapter07.md

```diff
@@ -413,8 +413,7 @@ Security certification should encompass the following elements:
 
 |  Ref | Requirement  | Definition/Note  |
 |-------|------|-------|
-| req.sec.sys.001 | The Platform **must** support authenticated and secure APIs, API endpoints. | |
-| | The Platform **must** implement authenticated and secure access to GUI. | |
```

Comment:
Suggest Changing this to the following and moving it to the end:

| req.sec.sys.015 | The Platform **must** implement authenticated and secure access to GUI. | |
|-------|------|-------|